lib/iex/lib/iex/helpers.ex

```diff
@@ -111,8 +112,10 @@ defmodule IEx.Helpers do
         true ->
           []
       end
-    apps |> Enum.reverse |> Enum.each(&Application.stop/1)
-    apps
+    not_started? =
+      Enum.reverse(apps)
```

Comment:
Hahaha. I hate when you have just a single pipe. :P Or you make it two, or you make it none. :P